packages/wdio-sync/src/index.ts

```diff
@@ -5,17 +5,35 @@ import runFnInFiberContext from './runFnInFiberContext'
 import wrapCommand from './wrapCommand'
 
 import { STACKTRACE_FILTER_FN } from './constants'
-const defaultRetries = { attempts: 0, limit: 0 }
+
+declare global {
+  namespace NodeJS {
+      interface Global {
+          _HAS_FIBER_CONTEXT: boolean
+          WDIO_WORKER: boolean
+          browser: {
+              _NOT_FIBER?: boolean
+          }
+      }
+  }
+}
+
+export type Retries = {
+    attempts: number;
+    limit: number;
+}
+
+const defaultRetries: Retries = { attempts: 0, limit: 0 }
```

Comment:
I'd avoid using this one as far as `retries` is now referencing the same object that would lead to unexpected behavior.

Reply:
I'm sorry, I don't quite understand, what do you mean exactly? Which unexpected behavior and referencing which same object?

Reply:
```suggestion
```

Reply:
If we remove this line, what would you do with `runSync` that uses `defaultRetries`? Replace it also by `{ attempts: 0, limit: 0 }` as we have in `executeSync` and go with having duplicated code?